youtube_dl/extractor/rtlnl.py

```diff
@@ -86,12 +86,12 @@ def _real_extract(self, url):
         # NB: nowadays, recent ffmpeg and avconv can handle these encrypted streams, so
         # this adaptive -> flash workaround is not required in general, but it also
         # allows bypassing georestriction therefore is retained for now.
-        videopath = material['videopath'].replace('/adaptive/', '/flash/')
```

Comment:
I forgot about the comment lines. As far as I'm concerned you remove all 3 of them, since they don't apply anymore.